server/controller/validation/course.validation.js

```diff
@@ -57,3 +58,34 @@ export const validateCourseId = [
     .isLength({ min: 24, max: 24 })
     .withMessage(ID_IS_INVALID),
 ];
+
+export const validateCourseQuery = [
```

Comment:
Validation for the new 'validateCourseQuery' validation array that the new API uses.